store/extensions/G-EnterFullRoom/extension.json

```
@@ -0,0 +1,44 @@
+{
+    "title": "G-EnterFullRoom",
+    "description": "Enter a full room automatically when someone leave.",
+    "authors": [
+      {
+        "name": "HappyBoyx3",
+        "hotel": ".de",
+        "username": "HappyBoyx3",
+        "discord": "MAIK#5094",
+        "twitter": "HabboMaik"
+      },
+	    
+	{		
+	"name": "JustProDylan",
+      }
+    ],
+    "version": "1.0",
+    "categories": ["Usability", "Others"],
+    
+    "source": "https://github.com/HabboMaik/G-EnterFullRoom",
+    "readme": "https://github.com/HabboMaik/G-EnterFullRoom/blob/main/README.md",
+    "releases": "https://github.com/HabboMaik/G-EnterFullRoom/releases/tag/v1.0"
+    
+    "stable": true,
+    
+    "framework": {
+      "name": "Geode",
+      "version": "1.5"
+    },

```

According to my decompiler the Geode version which was used is `1.4.1.0` not `1.5`

```suggestion
    "framework": {
      "name": "Geode",
      "version": "1.4.1.0"
    },
```